packages/components/src/components/search/_search.scss

```diff
@@ -72,6 +72,14 @@
     }
   }
 
+  .#{$prefix}--search-input:-webkit-autofill ~ .#{$prefix}--search-close {
+    fill: currentColor;
+  }
+
+  .#{$prefix}--search-input:-webkit-autofill ~ .#{$prefix}--search-close:hover {
+    background-color: #e5e5e5;
```

Comment:
Is it OK to make this non-themeable? Also, is there really no way to override `:-webkit-autofill`? Some sites seem to suggest adding box-shadow to override the color.

Reply:
There is! I wasn't sure if we would want to remove the autofill indicator or style around it, I wanted to propose a solution so designers could weigh in on the preferred workaround

Reply:
I see, thank you for explaining @abbeyhrt - It'll be a good discussion to see it with designers wrt what the (themable) background color of such state should be.

Reply:
For this specific solution, I think the autofill replacement background color might need to use the hex value since if you use any of the hover tokens, for example `$hover-field` it will look like this: 
<img width="549" alt="Screen Shot 2019-12-16 at 4 41 36 PM" src="https://user-images.githubusercontent.com/17281178/70949280-fe8f4300-2022-11ea-90ec-93cb60294f0d.png">
which is too dark, i think.  Since this only affects when the chrome autofill color is there, it shouldn't have any greater impact on the themes. I'm down to try other things though!  This is just what I came up with when looking for a solution 🙂